index.html

```diff
@@ -6632,39 +6632,42 @@ <h3>Issuance Date</h3>
         <h3>Proofs (Signatures)</h3>
 
         <p>
-The cryptographic mechanism used to prove that the information in a
+The securing mechanism used to prove that the information in a
 [=verifiable credential=] or [=verifiable presentation=] was not tampered
 with is called a <em>proof</em>. There are many types of cryptographic proofs
 including, but not limited to, digital signatures and zero-knowledge proofs. In
```

Comment:
It's unclear whether anything here applies to non-cryptographic proofs, which must exist, since there would otherwise be no need to say "types of cryptographic proofs" instead of "types of proofs". (Also, "proofs" in whichever of those phrases is kept should be made singular, "types of [cryptographic] proof".)

Compounding the above, the text that follows seems to be of two minds about whether "proofs" are always "cryptographic proofs", or "proofs" include both "cryptographic proofs" and "non-cryptographic proofs".

This paragraph _could_ include something like, "All proofs discussed here are cryptographic proofs," to allow what follows to just say "proofs" instead of any/all instances of "cryptographic proofs", *if* that's accurate.